Directly_import_images.md

```diff
@@ -0,0 +1,80 @@
+# In this tutorial we'll directly import images from publicly available online datasets into napari. 
+
+It's not always that we'd have all the data on our system. Maybe we need to view some data available on online platforms or maybe we are working as a team on github and we need to view some images that are not already downloaded on our system. 
+
+This tutorial helps us import those images directly into napari and cuts down a lot of unnecessary hassle. We have given codes along with explanations and some useful links in case somebody wants to explore more.
+
+For the purpose of example we have chose this publicly available [online dataset](https://idr.openmicroscopy.org/ "IDR from Open Microscopy"). Its the Image Data resource from Open Microscopy.
+
+Step 1 : Open this link and Find the datasdet you want to view.
+![Homepage](https://drive.google.com/file/d/1Y75_78QItfhWOGkxiOktWd7F7UZ2TSiy/view?usp=sharing)
+
+Step 2: Click on any dataset and you’ll be shifted to a viewport like below: 
+![Thumbnails](https://drive.google.com/file/d/18Mh-IkpkpdaQfJrfvIT6diXv01MzGLMz/view?usp=sharing)
+
+Step 3: Click on any image that you would want to view in napari 
+![Final Image]("https://drive.google.com/file/d/1BXnLnMbW85iaiCw9E0LQcFzngiqeSz97/view?usp=sharing")
+
+Step 4: Note down the image ID
+Step 5: Open Command Line and type in "napari"
+Step 6: Type or Copy-Paste the following lines of code into the IPython console
+
```

Comment:
I think it would be great to show people first the easy way, and then the way where we download the image to disk with requests.

The easiest way from ipython is:
```python
import napari
url = "https://idr.openmicroscopy.org/webgateway/render_image/9627732/0/0/"
napari.view_image(path=url)
```

This loads the image directly into RAM and displays it with napari. The image doesn't get copied to your hard drive.

But you might want to copy the image to your hard drive before looking at the data with napari. This is very useful if you are working on the road and don't always have a good internet connection. Downloading the image to your hard drive means you can work on it anytime.